script/test

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# script/test: Run the test suite for the application. Optionally pass in a path
+#              to an individual test file to run a single test.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ -n "$DEBUG" ]; then
+	set -x
+fi
+
+echo "==> Updating..."
+script/update
```

Comment:
Not a blocking comment, more an general observation: I'm not sure I entirely agree with the suggestion in scripts-to-rule-them-all that `script/test` "should probably call [script/update](https://github.com/github/scripts-to-rule-them-all/blob/master/script/update) to always ensure that the application is up to date".

I feel like it significantly slows down the speed of most `script/test` runs, particularly if you're only running a single test, to the extent that it's preferable to just call the test framework directly from the command line... which is the very thing that the pattern is meant to prevent you having to do! 

But like I say, not a request for a change, just a general moan about the approach they suggest :).